coalib/testing/LocalBearTestHelper.py

```diff
@@ -84,6 +84,7 @@ def check_results(self,
                       local_bear,
                       lines,
                       results,
+                      settings={},
```

Comment:
this one really has to be the last parameter so we don't break compatability ! :+1: